pkg/api/controller.go

```diff
@@ -49,6 +51,25 @@ func (c *Controller) Run() error {
 		os.Exit(1)
 	}
 
+	// Updating the CVE Database
+	if c.Config != nil && c.Config.Extensions != nil && c.Config.Extensions.Search != nil &&
+		c.Config.Extensions.Search.CVE.UpdateInterval > 1 {
```

Comment:
If we misconfigure the update interval to be <= 1, it looks like it'll just silently never start the updater, right?
Might be friendlier to have that as a separate 'if' so you can log a message that says e.g. "update interval config <= 1, not scheduling CVE DB updates.

In fact - unless this is covered elsewhere in the logging, It'd also be nice to have an else at the end to just log that it's not scheduling any CVE updates. It's nice to be explicit about this kind of thing in logs, so you don't have to look at code and think hard about what it was doing based on logs that should be there but aren't.

Reply:
1. I have added a log message where this IsValidImageFormat is called. 
2. Every time I add a new line to file looks like VS code removes it upon saving, I will add it now.
3. I will add that separate if condition and log a message but do we need to proceed further if updates are not scheduled.

Reply:
ok to 1 & 2 - for 3, I'm not sure I follow.
I was suggesting something roughly like this (but you can probably think of ways to improve this logic)

Also, these aren't blockers, if you don't see the value in adding these logs that's fine by me.

```go
// Updating the CVE Database
 	if c.Config != nil && c.Config.Extensions != nil && c.Config.Extensions.Search != nil {
 		if c.Config.Extensions.Search.CVE.UpdateInterval <= 1 {
				c.Log.Info().Msg("CVE update interval set to too-short interval <= 1, not scheduling updates.")
               }else{
		go func() {
 			for {
 				c.Log.Info().Msg("Updating the CVE database")
  				err := cveinfo.UpdateCVEDb(c.Config.Storage.RootDirectory, c.Log)
 				if err != nil {
 					panic(err)
 				}

  				c.Log.Info().Str("Db update completed, next update scheduled after ", c.Config.Extensions.Search.CVE.UpdateInterval.String()).Msg("hours") //nolint: lll
  				time.Sleep(c.Config.Extensions.Search.CVE.UpdateInterval * time.Hour)
 			}
 		}()
 	    }
    }else{
            c.Log.Info().Msg("No CVE config specified, not scheduling CVE updates")
}
```

So it doesn't proceed with scheduling anything if the interval is too short, it just logs it and moves on...

Reply:
I will update if-else logic as above and add this log messages.